databases/metabase/Portfile

```diff
@@ -0,0 +1,140 @@
+                    their data on their own.
+long_description    ${description}
+
+platforms           {darwin any}
+supported_archs     noarch
+license             MIT
+
+master_sites        https://downloads.metabase.com/v${version}/
+set metabase_jar    ${name}.jar
+distfiles           ${metabase_jar}
+
+checksums           rmd160  e9001956f7b0646411e77988d9ae539127bce136 \
+                    sha256  9bf03a464163971196bec306590041e8f571f24e1d5a34695363f0dea1eba809 \
+                    size    325002691
+
+java.version        11+
+java.fallback       openjdk11
+
+use_configure       no
+extract {}
+build {}
+test {}
+
+set metabase_wrapper     ${prefix}/etc/LaunchDaemons/org.macports.${name}/${name}.wrapper
+set metabase_exec_dir    ${prefix}/libexec/${name}
+set metabase_home_dir    ${prefix}/var/lib/${name}
+set metabase_log_dir     ${prefix}/var/log/${name}
+set metabase_run_dir     ${prefix}/var/run/${name}
+set metabase_db_dir      ${prefix}/var/db/${name}
+set metabase_exec        ${metabase_exec_dir}/${metabase_jar}
+set metabase_log         ${metabase_log_dir}/${name}.log
+set metabase_pid         ${metabase_run_dir}/${name}.pid
+set metabase_host        127.0.0.1
+set metabase_port        3000
+set metabase_user        metabase
+set metabase_group       metabase
+
+set metabase_dir_list [list \
+    ${metabase_exec_dir} \
+    ${metabase_home_dir} \
+    ${metabase_log_dir} \
+    ${metabase_run_dir} \
+    ${metabase_db_dir} \
+]
+
+set metabase_exec_list [list \
+    ${distpath}/${metabase_jar}
+]
+
+foreach d ${metabase_dir_list} {
+    destroot.keepdirs-append ${destroot}${d}
+}
+
+destroot {
+    foreach d ${metabase_dir_list} {
+        xinstall -d -m 0755 -g ${metabase_group} -o ${metabase_user} \
+            ${destroot}${d}
+    }
+
+    foreach f ${metabase_exec_list} {
+        xinstall -m 0644 -g ${metabase_group} -o ${metabase_user} \
+            ${f} \
+            ${destroot}${metabase_exec_dir}
+    }
+}
+
+post-deactivate {
+    # Cleanup all metabase-created files, which are significant
+    foreach d ${metabase_dir_list} {
+        if { [file exists ${d}] } {
+            ui_info "post-deactivate: dir exists, deleting: ${d}"
+            delete -force ${d}
+        } else {
+            ui_info "post-deactivate: dir doesn't exist, skipping delete: ${d}"
+        }
+    }
+}
+
+add_users               ${metabase_user} \
+                        group=${metabase_group} \
+                        home=${metabase_home_dir} \
+                        shell=/bin/sh \
+                        realname=metabase
+
+startupitem.create      yes
+
+# Enable additional logging, for diagnostic purposes
+startupitem.logevents   yes
+
+startupitem.user        ${metabase_user}
+startupitem.group       ${metabase_group}
+
+# Note: To avoid breakage when a Java port is updated/removed, use 'java_home' to select at runtime
+startupitem.init "
+export JAVA_HOME=\$(/usr/libexec/java_home)
```

Comment:
For `java_home`, it would be preferable to invoke that as follows (as we did for your PR for `jenkins`):

```
/usr/libexec/java_home --failfast --version ${java.version}
```

Unless there's a specific reason you did it this way...?

Reply:
The reason removed version check beause

> For `java_home`, it would be preferable to invoke that as follows (as we did for your PR for `jenkins`):
> 
> ```
> /usr/libexec/java_home --failfast --version ${java.version}
> ```
> 
> Unless there's a specific reason you did it this way...?

The reason for removing java version filter is that if only one version of java is installed, java_home will fail because the version cannot be found.

I will add a fallback to make it works and patch the jenkins port too.